src/core/lib/iomgr/ev_epoll1_linux.c

```diff
@@ -1074,11 +1092,13 @@ static grpc_error *pollset_kick(grpc_pollset *pollset,
           goto done;
         }
       } else {
+        GRPC_STATS_INC_POLLSET_KICKED_AGAIN(exec_ctx);
         GPR_ASSERT(next_worker->kick_state == KICKED);
         SET_KICK_STATE(next_worker, KICKED);
         goto done;
       }
     } else {
+      GRPC_STATS_INC_POLLSET_KICKED_AGAIN(exec_ctx);
```

Comment:
This should be GRPC_STATS_INC_POLLSET_KICK_OWN_THREAD

Reply:
Done